backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/queries.html.ftl

```diff
@@ -0,0 +1,322 @@
+window.onload = (event) => {
+    setInterval(getQueries, 10000);
+    runningQueriesTable = document.getElementById("runningQueriesTable");
+    notStartedQueriesTable = document.getElementById("notStartedQueriesTable");
+    failedQueriesTable = document.getElementById("failedQueriesTable");
+    succeedQueriesTable = document.getElementById("succeedQueriesTable");
+
+    runningNbrElt = document.getElementById("runningNbr");
+    notStartedNbrElt = document.getElementById("notStartedNbr");
+    failedNbrElt = document.getElementById("failedNbr");
+    succeedNbrElt = document.getElementById("succeedNbr");
+
+
+}
+
+function updateQueriesTable() {
+    clearTables();
+    queryCounter = 0;
+    updateQueriesInnerCounter();
+    for (i = 0; i < queries.length; i++) {
+        queryCounter++;
+        if(queries[i].status === "RUNNING")
+            runningQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+        if(queries[i].status === "NEW")
+            notStartedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+
+        if(queries[i].status === "FAILED")
+            failedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+
+
+        if(queries[i].status === "DONE")
+            succeedQueriesTable.insertAdjacentHTML('beforeend', getHtmltemplate(queries[i]));
+    }
+    if (k <= 30)
+        k++;
+    else k = 0;
+}
+
+function updateQueriesInnerCounter() {
+    runningNbrElt.innerText = queries.filter(x => x.status === "RUNNING").length;
+    notStartedNbrElt.innerText = queries.filter(x => x.status === "NEW").length;
+    failedNbrElt.innerText = queries.filter(x => x.status === "FAILED").length;
+    succeedNbrElt.innerText = queries.filter(x => x.status === "DONE").length;
+}
+
+function clearTables() {
+    for (var i = runningQueriesTable.rows.length - 1; i > 0; i--) {
+        runningQueriesTable.deleteRow(i);
+    }
+
+    for (var i = notStartedQueriesTable.rows.length - 1; i > 0; i--) {
+        notStartedQueriesTable.deleteRow(i);
+    }
+
+    for (var i = failedQueriesTable.rows.length - 1; i > 0; i--) {
+        failedQueriesTable.deleteRow(i);
+    }
+
+
+    for (var i = succeedQueriesTable.rows.length - 1; i > 0; i--) {
+        succeedQueriesTable.deleteRow(i);
+    }
+}
+
+function getQueries() {
+    try {
+        fetch(
+                '/admin/queries', {
+                    method: 'get',
+
+                    headers: {
+                        'Accept': 'application/json'
+                    },
+
+                    credentials: 'same-origin'
+                }).then(response => response.json()).then(data => {
+                queries = data;
+                updateQueriesTable();
+            })
+            .catch(error => {
+                console.log(error);
+            });
+
+    } catch (error) {
+        console.log(error);
+        return null;
+    };
+
+}
+<#noparse >
+    function getHtmltemplate(data) {
```

Comment:
Templateception :D
![yo dawg](https://imgflip.com/s/meme/Yo-Dawg-Heard-You.jpg)